src/templates/static/legal/complaints.jsx

```diff
@@ -2,25 +2,26 @@ import React from 'react';
             '<a href="https://idpc.org.mt/en/Pages/Home.aspx" target="_blank" rel="noopener noreferrer">', '</a>')}
         </p>
-        <p data-show='eucountry'>{it.L('If your complaint relates to an outcome of a trade or a transaction and remains unresolved, it will turn into a dispute. Should you be unsatisfied with our response, you can choose to escalate your complaint to the regulator or to an alternative dispute resolution entity.')}</p>
-        <p data-show='eucountry'>{it.L('Clients registered with Binary (IOM) Ltd can raise their unresolved disputes with the alternative dispute resolution entity IBAS by filling the adjudication form on the ADR entity\'s [_1]website[_2].',
+        <p>{it.L('Clients registered with Binary (IOM) Ltd can formally submit a complaint to their local Supervisory Authority.')}</p>
+        <p data-show='eucountry'>{it.L('If the client\'s complaint relates to an outcome of a trade or a transaction and remains unresolved, it will turn into a dispute. Should clients be unsatisfied with the Company\'s response, they can choose to escalate their complaint to the regulator or to an alternative dispute resolution entity.')}</p>
+        <p>{it.L('Clients registered with Binary (IOM) Ltd can raise their unresolved disputes with the alternative dispute resolution entity IBAS by filling the adjudication form on the ADR entity\'s [_1]website[_2].',
             '<a href=\'https://www.ibas-uk.com\' target=\'_blank\' rel=\'noopener noreferrer\'>', '</a>')}
         </p>
-        <p data-show='eucountry'>{it.L('Clients registered with Binary (Europe) Ltd can raise their unresolved disputes with the Malta Gaming Authority [_1] by submitting an email to [_2]. UK clients registered with Binary (Europe) Ltd can submit their dispute with the alternative dispute resolution entity IBAS by filling the adjudication form on the ADR entity\'s [_3]website[_4].',
-            '<a href="http://www.mga.org.mt/" target="_blank" rel="noopener noreferrer">(www.mga.org.mt)</a>', 'support.mga@mga.org.mt', '<a href=\'https://www.ibas-uk.com\' target=\'_blank\' rel=\'noopener noreferrer\'>', '</a>')}
+        <p data-show='eucountry'>{it.L('Clients registered with Binary (Europe) Ltd can raise their unresolved disputes with the alternative dispute resolution entity IBAS by filling the adjudication form on the ADR entity\'s [_1]website[_2] or they can refer to the Malta Gaming Authority via the [_3]Player Support Unit[_4].',
+            '<a href="https://www.ibas-uk.com" target="_blank" rel="noopener noreferrer">', '</a>', '<a href="http://www.mga.org.mt/support/online-gaming-support/" target=\'_blank\' rel=\'noopener noreferrer\'>', '</a>')}
```

Comment:
no need to repeat `</a>` for every link in a phrase, you can just use [_2] (or any index you have for it) as placeholder everywhere

Reply:
According to @negarn it is better to repeat it because in the past this confused the translate function.

Reply:
not the translate function, our translators :) they get confused when it's repeating placeholder numbers and think it's a mistake.

Reply:
noted, thanks :+1: